.env

```diff
@@ -81,5 +81,12 @@ SITE_NAME="Jira economics"
 SITE_URL="http://jira_economics.docker.localhost"
 # No lading slash!
 SITE_LOGO_PATH="build/images/itk-jira-symbol.svg"
-MAILER_NAME="Jira economics"
-MAILER_EMAIL="info@jira-economics.com"
+
+###> symfony/swiftmailer-bundle ###
+# For Gmail as a transport, use: "gmail://username:password@localhost"
+# For a generic SMTP server, use: "smtp://localhost:25?encryption=&auth_mode="
+# Delivery is disabled by default via "null://localhost"
+MAILER_NAME="ITK"
+MAILER_URL=smtp://mailhog:1025
+MAILER_EMAIL="info@support.srvitkphp72stg.itkdev.dk"
```

Comment:
Doesn't this belong in .env.local ?

Reply:
If you want values that applies to dev environment use .env.dev

Reply:
Replaced by empty strings